### PR TITLE
Ticket 36 : btn of create product

### PR DESCRIPTION
**Pull request by maximFeldman1**
create btn  and  clicking on it takes you to another url(page)
fixed: 
Screen Shot 2022-12-22 at 10 30 55

### CODE REVIEW

File: src/Components/Actions/CreateProductBtn/index.tsx [hunk 1]
```diff
@@ -1,5 +1,12 @@
 import React from "react";
+import "primeicons/primeicons.css";
+import { Link } from "react-router-dom";
+import { URL } from "../../../shared";
 
 export const CreateProductBtn = () => {
-  return <div>Button of create form </div>;
+  return (
+    <Link to={URL.CREATE}>
+      <i className="pi pi-plus-circle m-2"></i>
```
Comment (amitmarko) [inline src/Components/Actions/CreateProductBtn/index.tsx]:
```suggestion
      <i className="pi pi-plus-circle m-2" />
```

File: src/Components/App/AppRouter/index.tsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ const AppRouter = () => (
       <Routes>
         <Route path={URL.PRODUCTS} element={<Products />} />
         <Route path="/*" element={<Navigate to={URL.PRODUCTS} />} />
+        <Route path={URL.CREATE} element={<CreateForm />} />
```
Comment (amitmarko) [inline src/Components/App/AppRouter/index.tsx]: `*` It should be on the end

File: src/Pages/CreateProduct/index.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export const CreateForm = () => {
```
Comment (amitmarko) [inline src/Pages/CreateProduct/index.tsx:3]:
```suggestion
export const CreateProduct = () => {
```

File: src/shared/index.tsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 export const URL = {
   PRODUCTS: "/products",
+  CREATE: "/products/create",
```
Comment (amitmarko) [inline src/shared/index.tsx]:
```suggestion
  CREATE_PRODUCT: "/products/create",
```

**Review by maximFeldman1 — COMMENTED** [review]
Screen Shot 2022-12-22 at 11 08 04

**Review by amitmarko — APPROVED** [review]
LGTM

**Pull request merged**